src/plugin/index.js

```diff
@@ -94,4 +96,16 @@ Jarvis.prototype.apply = function(compiler) {
   // that's it!
 };
 
+const parseAuthor = function(author) {
+  if (typeof author === "string") {
+    const authorsArray = authors(author);
+    if (authorsArray.length > 0) {
+      return authorsArray[0];
+    }
+  } else if (author.name) {
+    return author;
+  }
+
+  return { name: "", email: "", url: "" };
```

Comment:
You can simplify this with a default argument

```javascript
const emptyAuthor  = { name: "", email: "", url: "" };
const parseAuthor = (author: emptyAuthor) => {
  if (typeof author === "string" && authors(author).length > 0) {
      return authors(author)[0];
  }
  return author;
}
```

Reply:
I will apply your feedback @Pickra !

Reply:
👍